src/python/pants/option/global_options.py

```diff
@@ -406,6 +406,9 @@ def register_bootstrap_options(cls, register):
                   'Because pantsd currently does not support parallel runs, '
                   'any prior running Pants command must be finished for the current one to start. '
                   'To never timeout, use the value -1.')
+    register('--pantsd-local-client-fallback', type=bool,
+             help='Whether to avoid waiting for exclusive access to pantsd in the case of '
+                   'concurrent pants invocations and just invoke pants without pantsd.')
```

Comment:
Might help to say start with something like `If true, avoid waiting for exclusive access..`